package.json

```diff
@@ -106,6 +106,45 @@
             "isSticky": true,
             "description": "Analyze a collection's schema."
           }
+        ],
+        "disambiguation": [
+          {
+            "category": "mongodb",
+            "description": "The user is asking for assistance or information related to MongoDB.",
+            "examples": [
+              "Explain this MongoDB query.",
+              "How do I write this MongoDB query in Python?",
+              "How do I use $lookup in MongoDB?",
+              "Can you help me debug a MongoDB query?",
+              "Rewrite this MongoDB query",
+              "How to convert this MongoDB query to Python?",
+              "How to convert this MongoDB query to Java?",
+              "How to convert this MongoDB query to Node.js?",
+              "How to convert this MongoDB query to Go?",
+              "How to convert this MongoDB query to php?",
+              "How do I update documents in MongoDB?",
+              "How do I insert documents in MongoDB?",
+              "How do I create documents in MongoDB?",
+              "How do I delete documents in MongoDB?",
+              "Explain how to use MongoDB transactions.",
+              "Help me write this MongoDB query:",
```

Comment:
Is `:` intentional here?

Reply:
Yes, figured some users might write prompts in that format when highlighting a specific code snippet.